### DISCUSSION TITLE
How to start mutiple wiki.js instances

### DISCUSSION THREAD

**Discussion by prozac64** [requarks/wiki#6851] [Help / Questions] [answered]
I would like tot start multiple instances of Wiki. js. Somehow this does not seem to be possible.
Any help would be greatly appreciated.

OS: Windows 2019 svr
Wiki.js: 2.5.300
Node.js: 20.9
PostgreSQL: 15.5

**Comment by NGPixel** [comment]
Either use different ports, or run multiple docker containers.

**Reply by prozac64** [reply]
Thnx for your quick response!

I have been using differnt ports 3000 and 3001.
The instance on port 3000 has been my first implementation of wiki.js. 

After building and starting the second instance i first get the installation finalisation screen, works all fine, but then after finishing it comes back up withe the "Service Unavailable" message.

Any thoughts?

**Comment by prozac64** [comment, marked as answer]
Solved it. Started the server by hand using [node server] command.
Turned out there was an issue witj [\node_modules\extract-files\package.json]

Copied the file from the first instance and all workes as expected.
